src/common/dateHelpers.js

```diff
@@ -52,6 +60,57 @@ const getRangedDateTime = granularity => {
   }
 };
 
+/**
+ * Generate a list of months for use in a select list.
+ *
+ * @param {string} month
+ * @returns {{keyDateTimeRanges: {}, listDateTimeRanges: *[]}|*}
+ */
+const getRangedMonthDateTime = month => {
+  const currentYear = Number.parseInt(moment.utc(getCurrentDate()).year(), 10);
+  const { startDate, endDate } = { ...rangedYearDateTime };
+  const keyDateTimeRanges = {};
+  let listDateTimeRanges = [];
+
+  const startDateUpdated = moment.utc(startDate);
+  const endDateUpdated = moment.utc(endDate);
+
+  while (endDateUpdated > startDateUpdated || startDateUpdated.format('M') === endDateUpdated.format('M')) {
+    const dateTime = {
+      value: {
+        startDate: startDateUpdated.toDate()
```

Comment:
note: we may want to consider just converting this directly into `toISOString` instead of offsetting that requirement to the components

we could also review using `toISOString` for the regular granularity ranges